.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
```

Comment:
How about making it HW Platform (iphone n, Pixel 2, intel based Mac, i7 based Dell with nVidia GPGPU etc., or Google Cloud machine), that way capturing the platform for Mobile as well HW.

Reply:
Okay sure